packages/little_logger/little_logger.0.3.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "A tiny, little logger <3"
+maintainer: "Ryan Moore"
+authors: "Ryan Moore"
+license: "MIT"
```

Comment:
The repo seem to have both MIT and Apache license. Do you want to have multiple entries here to match?

Reply:
Oh cool I didn't realize you could specify multiple licenses. I will remember that for next time, thanks. But I'm fine with leaving it as is for now.